documentation/controls.txt

```diff
@@ -0,0 +1,18 @@
+
+Driver:
+  Left joystick: controls speed/going forward
+    - y values
+  right joystick: controls turning
+    - x values
+  triggers: control gear shifting
+
+Operator:
```

Comment:
From Solomon:

Wing control should be the following:

- Left Wing: Y button for up, B button for down
- Right Wing: DPad Up for up, DPad Down for down

Wings have independent control of each other.

For the wing safety, Solomon would like the driver to be required to hold Y while the operator presses DPad Up or Y for the respective wings. This also sets wings up (thus releasing them).

Note that one wing can be "unlocked" while the other remains "locked". The driver still has to press Y to then unlock the other wing.

Reply:
I'm unsure if we should use these controls instead: this keeps us from needing a separate mode, which is probably better, which suggests we should do this one this way.

Reply:
done